config/filter.d/ignorecommands/apache-fakegooglebot.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Based on: https://isc.sans.edu/forums/diary/When+Google+isnt+Google/15968/
```

Comment:
well -- if based, then you would need first to clarify the copyright/license terms with original author.  Otherwise -- recode